### PR TITLE
userv/pipeline_server: add the pipeline server

### LINKED ISSUE #59: create a pipeline server to show the pipeline information

### PR DESCRIPTION
**Pull request by leyao-daily**
This PR is to fix #59, it will start a web server for use to check to demonstrate the pipeline server information.

### CODE REVIEW

File: cnap/userv/pipeline_server.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        Returns:
+            RuntimeDatabaseBase: An instance of the RuntimeDatabaseBase class.
+        """
+        if self._db is None:
+            runtime_db_type = _get_env("RUNTIME_DB_TYPE", "redis")
+            if runtime_db_type == "redis":
+                self._db = RedisDB()
+                redis_host = _get_env("REDIS_HOST", "127.0.0.1")
+                redis_port = _get_env("REDIS_PORT", 6379)
+                self._db.connect(redis_host, redis_port)
```
Comment (Hulongyin) [inline cnap/userv/pipeline_server.py:58]: Do we need to declare the exceptions that may be triggerred by `connect()` function in the docstring?

File: cnap/userv/pipeline_server.py [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+    @property
+    def pipelines(self) -> list:
+        """
+        Gets the pipeline list.
+
+        Returns:
+            list: List of pipeline dictionaries.
+        """
+        pipeline_dicts = self.db.get_all_table_objects_dict(PipelineService.PIPELINE_TABLE)
```
Comment (Hulongyin) [inline cnap/userv/pipeline_server.py:77]: Do we need to declare the exceptions that may be triggerred by `get_all_table_objects_dict()` function in the docstring?

File: cnap/userv/pipeline_server.py (hunk 1 above)
Comment (leyao-daily) [reply cnap/userv/pipeline_server.py:58]: Fixed.

**Review by Hulongyin — APPROVED** [review]
LGTM. Thanks

**Pull request merged**